src/v1/views/shared/Header.ts

```diff
@@ -154,9 +154,9 @@ export default class Header extends View {
           // Order of these calls is important for -
           // loader --> security/factor beacon swap.
           removeBeacon(this);
+          this.$el.removeClass(NO_BEACON_CLS);
```

Comment:
Need to remove class `NO_BEACON_CLS` anyway.
Otherwise beacon can have incorrect height after transition `loading -> null -> constroller beacon` (see videos)
In case of this transition `isLoading` (line 157) will be false in `swap` callback, but before `Animations.swapBeacons` it was true.
So to fix this height issue we can calculate `isLoading` before swap animation or remove `NO_BEACON_CLS ` class anyway.

Example on `/signin/verify/piv` 
Before this fix:

https://github.com/user-attachments/assets/ea925802-46b4-4117-af0e-7106b6613f9d


After:

https://github.com/user-attachments/assets/89aeefcb-e428-4891-8cb0-c8e8755b8566